flink-core/src/test/java/org/apache/flink/api/common/state/ListStateDescriptorTest.java

```diff
@@ -101,4 +108,35 @@ public void testValueStateDescriptorAutoSerializer() throws Exception {
 		assertNotNull(copy.getElementSerializer());
 		assertEquals(StringSerializer.INSTANCE, copy.getElementSerializer());
 	}
+
+	/**
+	 * FLINK-6775
+	 *
+	 * Tests that the returned serializer is duplicated if it is stateful. This allows to
```

Comment:
I think the wording could be a bit misleading, it's not only duplicated when it is stateful but it's always duplicated. (also holds for the other tests)